packages/grafana-prometheus/src/querycache/QueryCache.ts

```diff
@@ -120,10 +120,7 @@ export class QueryCache<T extends SupportedQueryTypes> {
       this.overlapWindowMs = durationToMilliseconds(duration);
     }
 
-    if (
-      (config.grafanaJavascriptAgent.enabled || config.featureToggles?.prometheusIncrementalQueryInstrumentation) &&
-      options.profileFunction !== undefined
-    ) {
+    if (config.grafanaJavascriptAgent.enabled && options.profileFunction !== undefined) {
```

Comment:
This will only run if faro is enabled, maybe we just should delete the whole profile function. The ff was only turned on in dev/stage

Reply:
https://github.com/grafana/deployment_tools/pull/78185
https://github.com/grafana/deployment_tools/pull/78186

Reply:
Then I'll remove it.

Reply:
Please see https://github.com/grafana/grafana/pull/89463/commits/51eb90f945d1ae0ac3bdbbb24342272199abdf65

Reply:
After merging this PR I'll clean the entries from `deployment_tools`